src/theme/components/button.ts

```diff
@@ -100,11 +103,12 @@ const pill = {
 const iconOnly = {
   bg: "ui.white",
   border: "1px solid",
-  borderColor: "ui.gray.light-cool",
-  color: "inherit",
+  borderColor: "ui.link.primary",
+  color: "ui.link.primary",
```

Comment:
Does inherit no longer work or is this an update for clarity?

Reply:
The color for this variant is changing altogether, so `inherit` will not work in this case.